From e92a630648d73850796db1c5fedece4937699378 Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Sun, 10 Jul 2011 12:57:12 -0400
Subject: drivers/sh: Add export.h for EXPORT_SYMBOL to intc/virq.c

This is exporting symbols and will fail to build once we remove
the implicit presence of module.h

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
(cherry picked from commit 07c92739455520541c22f6529683467aa3058b7c)

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 drivers/sh/intc/virq.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/sh/intc/virq.c b/drivers/sh/intc/virq.c
index 1e6e2d0..c7ec49f 100644
--- a/drivers/sh/intc/virq.c
+++ b/drivers/sh/intc/virq.c
@@ -14,6 +14,7 @@
 #include <linux/list.h>
 #include <linux/radix-tree.h>
 #include <linux/spinlock.h>
+#include <linux/export.h>
 #include "internals.h"
 
 static struct intc_map_entry intc_irq_xlate[NR_IRQS];
-- 
1.7.10

